core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -56,10 +56,10 @@ public class TaskManagerConfig
     private boolean shareIndexLoading;
     private int maxWorkerThreads = Runtime.getRuntime().availableProcessors() * 2;
     private Integer minDrivers;
-    private Integer initialSplitsPerNode;
+    private int initialSplitsPerNode = Runtime.getRuntime().availableProcessors() * 4;
     private int minDriversPerTask = 3;
     private int maxDriversPerTask = Integer.MAX_VALUE;
-    private Duration splitConcurrencyAdjustmentInterval = new Duration(100, TimeUnit.MILLISECONDS);
+    private Duration splitConcurrencyAdjustmentInterval = new Duration(1, TimeUnit.SECONDS);
```

Comment:
Should this be a separate commit ?
Please add rationale for changing this config to the PR description. It's not super obvious how this relates to assigning more splits

Reply:
I will extend description.

Reply:
Initial worker split concurrency is two times higher, but much slower to be adjusted (increased) based on memory utilization.

Reply:
> Initial worker split concurrency is two times higher, but much slower to be adjusted (increased) based on memory utilization.

It would be great to add a word why it helps. Even as top-level retro-post here in PR

Reply:
added to description